tests/integration/components/truncate-multiline-test.js

```diff
@@ -193,6 +193,64 @@ test('specifying different button texts works', function(assert) {
   );
 });
 
+test('specifying different button a11y texts works', function(assert) {
+  // Template block usage:
+  this.render(hbs`
+    <div style="width: 362px; font: 16px sans-serif;">
+      {{truncate-multiline seeMoreButtonText="click me" seeMoreButtonA11yText="Show more about this test" seeLessButtonText="then click me" seeLessButtonA11yText="Show less about this test" text="supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious"}}
+    </div>
+  `);
+
+  assert.equal(
+    this.$('.truncate-multiline--visually-hidden').text().trim(),
+    'Show more about this test',
+    'custom "see more" and a11y text is correct'
```

Comment:
This is no longer asserting that the custom "see more" text is correct. That's okay since it's already covered by another test, but the assertion message needs to be updated.